Modules/System/Cryptography Management/src/CryptographyManagementImpl.Codeunit.al

```diff
@@ -610,4 +611,78 @@ codeunit 1279 "Cryptography Management Impl."
 
         exit(Convert.ToBase64String(ByteArray));
     end;
+
```

Comment:
Just an organizing comment, could you move the code here to a new codeunit > "X509Certificate2 Cryptography Impl."